src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -294,21 +308,18 @@ protected object RegistrationMessage
             }
         }
 
-        /// <summary>
-        /// TBD
-        /// </summary>
+        /// <inheritdoc cref="ActorBase.PreStart"/>
```

Comment:
FYI: `docfx` does not support `inheritdoc` well, only basic support:
https://github.com/dotnet/docfx/issues/247